src/logic/tokens/store/model/token.ts

```diff
@@ -4,7 +4,7 @@ export type TokenProps = {
   address: string
```

Comment:
Same here, we have 3 different types to represent a Token. Is it possible to unify this?